Update click method to handle unicode targets in `if element is None` block

```python
target = <|strategy=xpath|value=//a[@class='delete' and contains(@data-confirm, '润ᩊ庲𥣌阞𡵗𧟥𤄅謩ʙ')]|>

    def click(self, target, wait_for_ajax=True,
              ajax_timeout=30, waiter_timeout=12, scroll=True):
        """Locate the element described by the ``target`` and click on it.
    
            :param tuple || WebElement target: Could be either locator that
                describes the element or element itself.
            :param wait_for_ajax: Flag that indicates if should wait for AJAX after
                clicking on the element
            :param ajax_timeout: The amount of time that wait_fox_ajax should wait.
                This will have effect if ``wait_fox_ajax`` parameter is ``True``.
            :param waiter_timeout: The amount of time that wait_until_element
                should wait. That value should be specified when non-default delay
                is needed (e.g. long run procedures)
            :param scroll: Decide whether scroll to element in case it is located
                out of the page
            :raise: UINoSuchElementError if the element could not be found.
    
            """
        if isinstance(target, (tuple, Locator)):
            element = self.wait_until_element(target, timeout=waiter_timeout)
        else:
            element = target
        if element is None:
            raise UINoSuchElementError(
                u'{0}: element {1} was not found while trying to click'
>               .format(type(self).__name__, str(target))
            )
E           UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 72: ordinal not in range(128)

robottelo/ui/base.py:639: UnicodeDecodeError
```

--- a/robottelo/ui/base.py
+++ b/robottelo/ui/base.py
@@ -635,7 +635,7 @@ class Base(object):
             element = target
         if element is None:
             raise UINoSuchElementError(
-                u'{0}: element {1} was not found while trying to click'
+                '{0}: element {1} was not found while trying to click'
                 .format(type(self).__name__, str(target))
             )
         # Required since from selenium 2.48.0. which makes Selenium more

